roles/neutron-data/tasks/main.yml

```diff
@@ -1,5 +1,4 @@
 ---
-  #FIXME do we need this on rhel?
```

Comment:
Yes.  bwm-ng is a backend for sysctl, disk, netstat and multiple other tools.
ipset is used on compute nodes by neutron.
ebtables is used by neutron